MaxText/train.py

```diff
@@ -460,7 +460,7 @@ def setup_train_loop(config):
 
   if config.using_pipeline_parallelism:
     # The vocab tensor(s) of shape [vocab, embed] (and transpose) are not sharded by stage
-    params_sharded_tolerance=0.1
+    params_sharded_tolerance=0.5
```

Comment:
Since the embedding table is unsharded, it's a fixed overhead regardless of the node count, so with enough nodes even this tolerance will be too low. Could we remove this check for now?

Reply:
Hmm I have to think about this formula more, the vocab tensor is still sharded by FSDP and TP so there shouldn't be so many more params per chip with mix of PP + FSDP vs pure FSDP. I like the idea of removing this check completely though for now.